pennylane/debugging.py

```diff
@@ -259,15 +424,57 @@ def expval(op):
 
 
 def probs(wires=None, op=None):
-    """Compute the probability distribution for the state.
+    """Compute the probability distribution for the state at the current
```

Comment:
Should we add a "see also" qml.probs?

Reply:
I think it might cause more confusion, since they are used in very different contexts.